setup.sh

```diff
@@ -5,17 +5,14 @@
 
 set -e
 
+# Docker environment which we will need to install for docker-osx-dev to work.
+export   DOCKER_HOST="tcp://localhost:2375"
```

Comment:
We could just set it without exporting.  Making it "readonly" makes it impossible to unset later inside env_is_defined() though.